[JIT] Include conv3d in the conv-add-relu fusion

desertfire opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#54772 [JIT] Include conv3d in the conv-add-relu fusion**

Summary: conv3d-add-relu fusion does not work on some platforms when TF32 is enabled, so set allow_tf32 to false.

Test Plan:
```
python test/test_jit.py -k test_freeze_conv_relu_fusion 
```

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D27435560](https://our.internmc.facebook.com/intern/diff/D27435560)

desertfire committed:
[JIT] Include conv3d in the conv-add-relu fusion

Summary:

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

desertfire committed:
Update on "[JIT] Include conv3d in the conv-add-relu fusion"


Summary: Previously two CI tests failed after replacing cond3d-add-relu

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

desertfire committed:
Update on "[JIT] Include conv3d in the conv-add-relu fusion"


Summary:

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

eellison reviewed (commented):
Can we only disable tf32 if we're running conv3d ?

ngimel commented:
Can you please file a separate issue for tf32 failure (with instructions to repro)? It should not fail.

desertfire committed:
Update on "[JIT] Include conv3d in the conv-add-relu fusion"


Summary: conv3d-add-relu fusion does not work on some platforms when TF32 is enabled, so set allow_tf32 to false.

Test Plan:
```
python test/test_jit.py -k test_freeze_conv_relu_fusion 
```

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D27435560](https://our.internmc.facebook.com/intern/diff/D27435560)

[ghstack-poisoned]

desertfire committed:
Update on "[JIT] Include conv3d in the conv-add-relu fusion"


Summary: conv3d-add-relu fusion does not work on some platforms when TF32 is enabled, so set allow_tf32 to false.

Test Plan:
```
python test/test_jit.py -k test_freeze_conv_relu_fusion 
```

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D27435560](https://our.internmc.facebook.com/intern/diff/D27435560)

[ghstack-poisoned]

desertfire committed:
Update on "[JIT] Include conv3d in the conv-add-relu fusion"


Summary: conv3d-add-relu fusion does not work on some platforms when TF32 is enabled, so set allow_tf32 to false.

Test Plan:
```
python test/test_jit.py -k test_freeze_conv_relu_fusion 
```

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D27435560](https://our.internmc.facebook.com/intern/diff/D27435560)

[ghstack-poisoned]

eellison reviewed (approved):
nice!
